website/source/docs/providers/google/r/sql_database_instance.html.markdown

```diff
@@ -0,0 +1,146 @@
+---
+layout: "google"
+page_title: "Google: google_sql_database_instance"
+sidebar_current: "docs-google-sql-database-instance"
+description: |-
+  Creates a new SQL database instance in Google Cloud SQL.
+---
+
+# google\_sql\_database\_instance
+
+Creates a new Google SQL Database Instance. For more information, see the [official documentation](https://cloud.google.com/sql/), or the [JSON API](https://cloud.google.com/sql/docs/admin-api/v1beta4/instances).
+
+## Example Usage
+
+Example creating a SQL Database.
+
+```
+resource "google_sql_database_instance" "master" {
+    name = "master-instance"
+
+    settings {
+        tier = "D0"
+    }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the instance.
+
+* `region` - (Optional) The region the instance will sit in. Defaults to the
```

Comment:
Update this to make it clear that the Cloud SQL regions are different to the Compute Engine regions, and maybe link to a list of these regions in the Cloud SQL documentation.